src/cnguts.cpp

```diff
@@ -34,8 +34,9 @@
 #include <cnewin.h>		// BrHans -- if needed here
 #endif
 #endif
+#ifdef BINRES //cnglob.h
```

Comment:
This is where you want to say:

```
// TODO (MP) Windows io header was replaced for C-style file io fopen.fclose, etc. These changes were not tested.
```

I assume the `//cnglob.h` is supposed to indicate this is defined (or not) in cnglob.h. But we had also talked about making BINRES a CMake option. Where did we land on that? I think I have a slight preference to make it a CMake option (then we can include the TODO (MP) there).